pkg/specgenutil/specgen.go

```diff
@@ -737,8 +737,8 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 		case "proc-opts":
 			s.ProcOpts = strings.Split(val, ",")
 		case "seccomp":
-			s.SeccompProfilePath = val
-			s.Annotations[define.InspectAnnotationSeccomp] = val
+			s.SeccompProfilePath, _ = specgen.ConvertWinMountPath(val)
+			s.Annotations[define.InspectAnnotationSeccomp], _ = specgen.ConvertWinMountPath(val)
```

Comment:
The error should not be ignored?! It would be very wrong to ignore invalid input and just unset the profile path.

Reply:
It's ok to ignore the error because `ConvertWinMountPath` still returns [the original path](https://github.com/l0rd/podman/blob/8fd7f4fbc9885ddc19ece95f2537e20fabc3306a/pkg/specgen/winpath.go#L55), it doesn't unset it.